unyt/_unit_lookup_table.py

```diff
@@ -428,6 +429,8 @@
         ),
         # misc
         ("eV", (J_per_eV, dimensions.energy, 0.0, r"\rm{eV}", True)),
+        ("foe", (J_per_foe, dimensions.energy, 0.0, r"\rm{foe}", False)),
+        ("bethe", (J_per_foe, dimensions.energy, 0.0, r"\rm{B}", False)),
```

Comment:
This is correct, but *looks* like a mistake. Can you add a comment to reassure future readers that yes, 1 foe = 1 bethe ?

Reply:
comment added